config/config.go

```diff
@@ -83,6 +84,8 @@ var (
 	agentConfigMx sync.RWMutex
 	version       string
 	lEtag         = &lastEtag{Etag: "0"}
+
+	capabilities = []string{"PATCH_GA", "GUEST_POLICY_BETA"}
```

Comment:
Maybe have a comment on how these values are decided?

Reply:
Made a comment for how they are used, i don't really want to publish how we decide them beyond this.